app/controllers/home_controller.rb

```diff
@@ -38,9 +40,10 @@ def static
 
   def unsubscribe
     code = params[:code]
-    User.all.select {|x| x.encrypted_email == code }.each {|x| x.update_attributes subscribed_to_newsletter: false }
-    flash[:message] = "You have been unsubscribed from the newsletter."
-    redirect_to "/anime"
+    User.all.select { |x| x.encrypted_email == code }
+      .each { |x| x.update_attributes subscribed_to_newsletter: false }
```

Comment:
Yeah looks like that's the case. Maybe this should be changed but I think I don't know enough about your code and Rails yet to do that.